apps/emqx/test/emqx_sys_mon_SUITE.erl

```diff
@@ -80,14 +80,25 @@ end_per_testcase(_, _Config) ->
     emqx_common_test_helpers:stop_apps([]).
 
 t_procinfo(_) ->
-    ok = meck:new(emqx_vm, [passthrough, no_history]),
-    ok = meck:expect(emqx_vm, get_process_info, fun(_) -> [] end),
-    ok = meck:expect(emqx_vm, get_process_gc_info, fun(_) -> [] end),
-    ?assertEqual([{pid, undefined}], emqx_sys_mon:procinfo(undefined)),
```

Comment:
This case is impossible: if `procinfo` receives anything but a PID, it'll crash because `emqx_vm:get_process_info/1` has a guard that only accepts PIDs.